cfme/containers/image.py

```diff
@@ -166,6 +166,8 @@ class Details(CFMENavigateStep):
     prerequisite = NavigateToAttribute("parent", 'All')
 
     def step(self):
+        self.prerequisite_view.entities.get_entity(provider=self.obj.provider.name,
```

Comment:
This should replace the call on L171, shouldn't it?  If so it needs the click call as well. Right now this call returns an entity, which isn't captured, and then calls get_entity again, not using search, and calls click().

Reply:
On Container Images table , the image name is not a unique key.
There for we use the search by image name (the only possibility that we have to use search by name),In order to reduce the number of lines on table , and get view with all entity's that includes this name.
Then we use the get_entity with Image ID that is the unique key , in order to select the specific image and click on it.
So the get_entity is not duplicated in this special condition that we have.

Reply:
@brahmanim, did you try to pass both name and id ? It should work. Please let me know if it doesn't work. I'll fix this.